Armstrong_Number/Armstrong_Number.coffee

```diff
@@ -0,0 +1,30 @@
+armstrongNumber = (num) -> 
+  originalnum = num
+  reoriginalnum=num
+  
+  result=0
+  count=0
+  while num != 0
+    num = parseInt(num / 10)
+    count++
+   
+  while reoriginalnum != 0
+    remainder = reoriginalnum % 10
+    result += Math.pow(remainder, count)
+    reoriginalnum = parseInt(reoriginalnum / 10)
+
+  
+  console.log result
```

Comment:
No need to print these many things. Just print the num and if it Armstrong number or not.